mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulGameScreen.kt

```diff
@@ -12,27 +12,24 @@ import ch.epfl.sdp.mobile.ui.game.GameScreen
 import ch.epfl.sdp.mobile.ui.game.GameScreenState
 import ch.epfl.sdp.mobile.ui.game.Move
 import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.launch
 
 /**
  * The [StatefulGameScreen] to be used for the Navigation
  *
  * @param user the currently logged-in user.
+ * @param id the identifier for the match.
  * @param modifier the [Modifier] for the composable
  */
 @Composable
 fun StatefulGameScreen(
     user: AuthenticatedUser,
+    id: String,
     modifier: Modifier = Modifier,
 ) {
-  val chessFacade = LocalChessFacade.current
+  val facade = LocalChessFacade.current
```

Comment:
Any reason for this renaming? I think it makes it a bit less clear.

Reply:
There's only a single facade used in this composable, so I thought would be easier to read since `facade` is slightly less verbose.